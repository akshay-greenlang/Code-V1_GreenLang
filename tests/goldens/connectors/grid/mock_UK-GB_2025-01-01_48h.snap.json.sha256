81ed314cbad88ae94efaf60155b3e12459b98b3f74ee41dec10c6af6135e5ba5