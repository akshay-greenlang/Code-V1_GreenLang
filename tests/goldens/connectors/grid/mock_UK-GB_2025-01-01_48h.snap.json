{"connector_id":"grid/intensity/mock","connector_version":"0.1.0","payload":{"metadata":{"algorithm":"seasonal_sinusoid","connector":"grid/intensity/mock","data_points":48,"region_offset":145,"seed":"c2c2e0eb7c2b10a9a4a42d12fc0e9fbc99090572ac2219f6b0fd0c60a8bf6241","version":"0.1.0"},"region":"UK-GB","resolution":"hour","series":[{"quality":"simulated","ts":"2025-01-01T00:00:00Z","unit":"gCO2/kWh","value":"445"},{"quality":"simulated","ts":"2025-01-01T01:00:00Z","unit":"gCO2/kWh","value":"496.8"},{"quality":"simulated","ts":"2025-01-01T02:00:00Z","unit":"gCO2/kWh","value":"545"},{"quality":"simulated","ts":"2025-01-01T03:00:00Z","unit":"gCO2/kWh","value":"586.4"},{"quality":"simulated","ts":"2025-01-01T04:00:00Z","unit":"gCO2/kWh","value":"618.2"},{"quality":"simulated","ts":"2025-01-01T05:00:00Z","unit":"gCO2/kWh","value":"638.2"},{"quality":"simulated","ts":"2025-01-01T06:00:00Z","unit":"gCO2/kWh","value":"645"},{"quality":"simulated","ts":"2025-01-01T07:00:00Z","unit":"gCO2/kWh","value":"638.2"},{"quality":"simulated","ts":"2025-01-01T08:00:00Z","unit":"gCO2/kWh","value":"618.2"},{"quality":"simulated","ts":"2025-01-01T09:00:00Z","unit":"gCO2/kWh","value":"586.4"},{"quality":"simulated","ts":"2025-01-01T10:00:00Z","unit":"gCO2/kWh","value":"545"},{"quality":"simulated","ts":"2025-01-01T11:00:00Z","unit":"gCO2/kWh","value":"496.8"},{"quality":"simulated","ts":"2025-01-01T12:00:00Z","unit":"gCO2/kWh","value":"445"},{"quality":"simulated","ts":"2025-01-01T13:00:00Z","unit":"gCO2/kWh","value":"393.2"},{"quality":"simulated","ts":"2025-01-01T14:00:00Z","unit":"gCO2/kWh","value":"345"},{"quality":"simulated","ts":"2025-01-01T15:00:00Z","unit":"gCO2/kWh","value":"303.6"},{"quality":"simulated","ts":"2025-01-01T16:00:00Z","unit":"gCO2/kWh","value":"271.8"},{"quality":"simulated","ts":"2025-01-01T17:00:00Z","unit":"gCO2/kWh","value":"251.8"},{"quality":"simulated","ts":"2025-01-01T18:00:00Z","unit":"gCO2/kWh","value":"245"},{"quality":"simulated","ts":"2025-01-01T19:00:00Z","unit":"gCO2/kWh","value":"251.8"},{"quality":"simulated","ts":"2025-01-01T20:00:00Z","unit":"gCO2/kWh","value":"271.8"},{"quality":"simulated","ts":"2025-01-01T21:00:00Z","unit":"gCO2/kWh","value":"303.6"},{"quality":"simulated","ts":"2025-01-01T22:00:00Z","unit":"gCO2/kWh","value":"345"},{"quality":"simulated","ts":"2025-01-01T23:00:00Z","unit":"gCO2/kWh","value":"393.2"},{"quality":"simulated","ts":"2025-01-02T00:00:00Z","unit":"gCO2/kWh","value":"445"},{"quality":"simulated","ts":"2025-01-02T01:00:00Z","unit":"gCO2/kWh","value":"496.8"},{"quality":"simulated","ts":"2025-01-02T02:00:00Z","unit":"gCO2/kWh","value":"545"},{"quality":"simulated","ts":"2025-01-02T03:00:00Z","unit":"gCO2/kWh","value":"586.4"},{"quality":"simulated","ts":"2025-01-02T04:00:00Z","unit":"gCO2/kWh","value":"618.2"},{"quality":"simulated","ts":"2025-01-02T05:00:00Z","unit":"gCO2/kWh","value":"638.2"},{"quality":"simulated","ts":"2025-01-02T06:00:00Z","unit":"gCO2/kWh","value":"645"},{"quality":"simulated","ts":"2025-01-02T07:00:00Z","unit":"gCO2/kWh","value":"638.2"},{"quality":"simulated","ts":"2025-01-02T08:00:00Z","unit":"gCO2/kWh","value":"618.2"},{"quality":"simulated","ts":"2025-01-02T09:00:00Z","unit":"gCO2/kWh","value":"586.4"},{"quality":"simulated","ts":"2025-01-02T10:00:00Z","unit":"gCO2/kWh","value":"545"},{"quality":"simulated","ts":"2025-01-02T11:00:00Z","unit":"gCO2/kWh","value":"496.8"},{"quality":"simulated","ts":"2025-01-02T12:00:00Z","unit":"gCO2/kWh","value":"445"},{"quality":"simulated","ts":"2025-01-02T13:00:00Z","unit":"gCO2/kWh","value":"393.2"},{"quality":"simulated","ts":"2025-01-02T14:00:00Z","unit":"gCO2/kWh","value":"345"},{"quality":"simulated","ts":"2025-01-02T15:00:00Z","unit":"gCO2/kWh","value":"303.6"},{"quality":"simulated","ts":"2025-01-02T16:00:00Z","unit":"gCO2/kWh","value":"271.8"},{"quality":"simulated","ts":"2025-01-02T17:00:00Z","unit":"gCO2/kWh","value":"251.8"},{"quality":"simulated","ts":"2025-01-02T18:00:00Z","unit":"gCO2/kWh","value":"245"},{"quality":"simulated","ts":"2025-01-02T19:00:00Z","unit":"gCO2/kWh","value":"251.8"},{"quality":"simulated","ts":"2025-01-02T20:00:00Z","unit":"gCO2/kWh","value":"271.8"},{"quality":"simulated","ts":"2025-01-02T21:00:00Z","unit":"gCO2/kWh","value":"303.6"},{"quality":"simulated","ts":"2025-01-02T22:00:00Z","unit":"gCO2/kWh","value":"345"},{"quality":"simulated","ts":"2025-01-02T23:00:00Z","unit":"gCO2/kWh","value":"393.2"}],"unit":"gCO2/kWh"},"provenance":{"connector_id":"grid/intensity/mock","connector_version":"0.1.0","metadata":{"data_points":48,"end":"2025-01-03T00:00:00+00:00","region":"UK-GB","resolution":"hour","start":"2025-01-01T00:00:00+00:00"},"mode":"record","query_hash":"4d482564b072b864775230ceb8bddb080eedd10a9a96fa24a8e49e496242213b","schema_hash":"2cf663a8a03d50acb0c1df2fba40940ffb4f63f0d1882a6386a70320e2bc06f6","seed":"c2c2e0eb7c2b10a9a4a42d12fc0e9fbc99090572ac2219f6b0fd0c60a8bf6241","snapshot_id":"d5d91fa8866ee3f2b71dbdd035d5b51c56f4a7d257e59f4b6d2cf6216b57f75d","timestamp":"2025-10-09T05:13:40.758896Z"},"schema":{"payload_model":"GridIntensityPayload","provenance_model":"ConnectorProvenance"}}